metals/src/main/scala/scala/meta/internal/metals/BuildServerConnection.scala

```diff
@@ -160,10 +159,34 @@ class BuildServerConnection private (
     register(server => server.workspaceBuildTargets()).asScala
   }
 
+  def buildTargetJavacOptions(
+      params: JavacOptionsParams
+  ): Future[JavacOptionsResult] = {
+    val resultOnJavacOptionsUnsupported = new JavacOptionsResult(
+      List.empty[JavacOptionsItem].asJava
+    )
+    if (isSbt) Future.successful(resultOnJavacOptionsUnsupported)
```

Comment:
@Arthurm1 I added this check since it seems that sbt shuts down when unexpected request is made.